ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -650,34 +677,25 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
               <div style={{display: 'flex', flexDirection: 'column'}}>
               <div style={{display: 'flex', alignItems: 'flex-end'}}>
                 <div style={styles.previewDataHeader}>
-                  Preview Data Set
+                  <div>
+                    <CircleWithText text={4} width='23.78px' height='23.78px'
+                      style={{marignTop: '0.3rem', fill: colorWithWhiteness(colors.primary, 0.5)}}/>
+                  </div>
+                  <label style={{marginLeft: '0.5rem', color: colors.primary}}>
+                    Preview Data Set
+                  </label>
+                </div>
+                <div style={{color: colors.primary, fontSize: '14px', width: '60%'}}>
+                  A visualization of your data table based on concept sets
+                  and values you selected above. Once complete, export for analysis
                 </div>
-                <Clickable data-test-id='refresh-preview-clickable-text'
-                           disabled={this.isRefreshPreviewDisabled}
-                           onClick={() => this.getPreviewList()}
-                           style={{fontSize: '12px',
-                             cursor: this.isRefreshPreviewDisabled ? 'not-allowed' : 'pointer',
-                             fontWeight: 600, lineHeight: '15px',
-                             color: this.isRefreshPreviewDisabled ?
-                               colorWithWhiteness(colors.dark, 0.6) : colors.accent
-                           }}>
-                    Refresh Preview
-                  </Clickable>
-              </div>
-              <div style={{color: colors.primary, fontSize: '14px'}}>
-                A visualization of your data table based on the variable
-                and value you selected above
               </div>
               </div>
-              <TooltipTrigger data-test-id='save-tooltip'
-                           content='Requires Owner or Writer permission' disabled={this.canWrite}>
-                <Button data-test-id='save-button'
-                  onClick ={() => this.setState({openSaveModal: true})}
-                  disabled={this.disableSave() || !this.canWrite}>
-                  {this.editing ? !(dataSetTouched && this.canWrite) ? 'Analyze' :
-                    'Update And Analyze' : 'Save And Analyze'}
-                </Button>
-              </TooltipTrigger>
+              <Clickable data-test-id='preview-button' style={{marginTop: '0.5rem',
+                height: '1.8rem', width: '6.5rem', color: colors.secondary}}
+                onClick={() => this.getPreviewList()}>
```

Comment:
I'm confused - what happens now if you click this without selecting anything? It shows an error modal? This seems worse than the current behavior